Makefile

```diff
@@ -46,4 +46,14 @@ check_trufflehog:
 trufflehog: check_trufflehog
 	trufflehog --regex --rules .trufflehog.json --entropy=False file:///$$azul_home
 
-.PHONY: all hello terraform deploy subscribe everything reindex clean test travis integration_test trufflehog check_trufflehog
+demolish: check_branch_personal
```

Comment:
Remove both. My apologies for wavering on this.

Reply:
Does that mean also remove the changes to the check_branch script?